conf/hosts.txt

```diff
@@ -1,8 +1,14 @@
 #This is the hosts file for Jasminegraph. All the host names on which Jasminegraph workers should get started needs
 #to be listed here
 localhost
+
+# TODO: Replace this
```

Comment:
Not clear what you mean by `Replace this`, Please provide more details and the context information so that this comment will help the reader to understand what action(s) to be taken.

Reply:
Removed that file. It is only used in native mode